CHANGES.rst

```diff
@@ -10,7 +11,7 @@
 2.0.0a3 (2015-12-03)
 ====================
 
-- [NEW] Added API documentation hosted on readthedocs.org.
+- [NEW] Added API documentation hosted on readthedocs.com.
```

Comment:
I'm confused by this change, AFAICT the documentation is on readthedocs.org not .com

Even if this change is correct it should be done separately to the rest of the work (I think a commit pushed to master would be fine). That way at least if for some reason the query PR was reversed or whatever at least this doc fix would still be in place.